recipe/gen_patch_json.py

```diff
@@ -368,6 +369,20 @@ def _fix_libgfortran(fn, record):
             record['depends'] = depends
 
 
+def _add_osx_virt(fn, record):
+    rconst = record.get("run_constrained", ())
+    dep_idx = next(
+        (q for q, dep in enumerate(rconst)
+         if dep.split(' ')[0] == "__osx"),
+        None
+    )
+    run_constrained = list(rconst)
+    if dep_idx is None:
+        run_constrained.append("__osx >=10.9")
```

Comment:
We don't need to add the 10.9 as it's implied by the miniconda installer. We should add this constraint for the few feedstocks that use one that's greater than 10.9

Reply:
ahhhh crap. I will redo the PR.